### PR TITLE
Add Wavehash Chain

### PR DESCRIPTION
**Pull request by ahmad-blocksdev**

### CODE REVIEW

File: testnets/wavehashtestnet/assetslist.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            "denom": "wahax",
+            "exponent": 6
+          }
+        ],
+        "base": "uwaha",
+        "name": "WaveHash",
+        "display": "wavehash",
+        "symbol": "WAHAX",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/WaveHashProtocol/chain-registry/master/testnets/wavehashtestnet/images/WaveHash.png",
```
Comment (JeremyParish69) [inline testnets/wavehashtestnet/assetslist.json:22]: Please correct these image URLs to point to the Cosmos org's Chain Registry

File: testnets/wavehashtestnet/assetslist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          {
+            "denom": "uwahax",
+            "exponent": 0
+          },
+          {
+            "denom": "wahax",
+            "exponent": 6
+          }
+        ],
+        "base": "uwaha",
```
Comment (JeremyParish69) [inline testnets/wavehashtestnet/assetslist.json:17]: is the base unit uwahax or uwaha?

File: testnets/wavehashtestnet/assetslist.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "name": "WaveHash",
+        "display": "wavehash",
```
Comment (JeremyParish69) [inline testnets/wavehashtestnet/assetslist.json:19]: display value must be one of the defined denom_units, so either uwahax or wahax (typically will be the larger)

File: testnets/wavehashtestnet/chain.json [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      },
+      "cosmwasm_version": "0.30",
+      "cosmwasm_enabled": true,
+      "ibc_go_version": "4.3.0",
+      "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/WaveHashProtocol/testnet/main/testnet-1/genesis.json"
+      },
+      "versions": [
+        {
+          "name": "v6.0.0",
```
Comment (JeremyParish69) [inline testnets/wavehashtestnet/chain.json:51]: you seem to have v6.0.0 in the versions array twice. Please only have one object for v6.0.0 (but you may define previous versions here)

File: testnets/wavehashtestnet/assetslist.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "symbol": "WAHAX",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/WaveHashProtocol/chain-registry/master/testnets/wavehashtestnet/images/WaveHash.png",
+          "svg": "https://raw.githubusercontent.com/WaveHashProtocol/chain-registry/master/testnets/wavehashtestnet/images/WaveHash.svg"
```
Comment (JeremyParish69) [inline testnets/wavehashtestnet/assetslist.json:23]: the svg seems to be a tiled conversion from the PNG--probably not worth including, in my opinion

**Comment by ahmad-blocksdev** [comment]
Resolved mentioned issues

**Pull request merged**